src/main/java/com/gamingmesh/jobs/listeners/JobsPaymentListener.java

```diff
@@ -239,8 +248,8 @@ public void onCowMilking(PlayerInteractEntityEvent event) {
 	}
 
 	if (Jobs.getGCManager().CowMilkingTimer > 0) {
-	    if (cow.hasMetadata(cowMetadata)) {
-		long time = cow.getMetadata(cowMetadata).get(0).asLong();
+	    if (cowMilkingTimer.getIfPresent(cowUUID) != null) {
+		long time = cowMilkingTimer.getIfPresent(cowUUID);

```

Comment:
Declare and cache `getIfPresent` to `Long` to make sure the process is fast enough, as `Map#get` is a bit slow.
```suggestion
	    Long time = cowMilkingTimer.getIfPresent(cowUUID);
	    if (time != null) {
```